<!-- -->
<launch>
    <param name ="/use_sim_time" value="false"/>

    <arg name="data_root" default="/media/bingo/SSD/multi_lidar_calib_data/sharingVAN/20200706/lidar_calibration" />

    <arg name="lidar_front_path" default="$(arg data_root)/lidar_front" />
    <arg name="lidar_left_path" default="$(arg data_root)/lidar_left" />
    <arg name="lidar_right_path" default="$(arg data_root)/lidar_right" />
    <arg name="lidar_back_path" default="$(arg data_root)/lidar_back" />

    <arg name="lidar_front_topic" default="/lidar2/points_raw" />
    <arg name="lidar_left_topic" default="/lidar1/points_raw" />
    <arg name="lidar_right_topic" default="/lidar3/points_raw" />
    <arg name="lidar_back_topic" default="/lidar4/points_raw" />

    <arg name="start_index" default="1" />

    <node pkg="multi_lidar_split_save" type="multi_lidar_split_save_four" name="multi_lidar_split_save_four" output="screen">
        <param name="lidar_front_path" value="$(arg lidar_front_path)" />
        <param name="lidar_left_path" value="$(arg lidar_left_path)" />
        <param name="lidar_right_path" value="$(arg lidar_right_path)" />
        <param name="lidar_back_path" value="$(arg lidar_back_path)" />

        <param name="lidar_front_topic" value="$(arg lidar_front_topic)" />
        <param name="lidar_left_topic" value="$(arg lidar_left_topic)" />
        <param name="lidar_right_topic" value="$(arg lidar_right_topic)" />
        <param name="lidar_back_topic" value="$(arg lidar_back_topic)" />

        <param name="start_index" value="$(arg start_index)" />

    </node>

</launch>
